src/DataGathering/webscraper.py

```diff
@@ -0,0 +1,15 @@
+class WebScraper:
```

Comment:
This entire directory was a fake for the documentation branch. It should have been deleted and should not be merged down.